public/index.html

```diff
@@ -15,6 +15,7 @@
     -->
     <link href="https://fonts.googleapis.com/css?family=Open+Sans|Poppins|Raleway|Oswald|Inter|Nunito:wght@100;200;300;400;500;600;700;800;900|Inter:wght@100;200;300;400;500;600;700;800;900|Lato&display=swap" rel="stylesheet">
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="%PUBLIC_URL%/style/graphql.css" rel="stylesheet" />
```

Comment:
It seems like its adding 1700 lines of code on every page load. Is there a way to change it?